browser/brave_wallet/asset_discovery_manager_unittest.cc

```diff
@@ -204,7 +204,7 @@ TEST_F(AssetDiscoveryManagerUnitTest, GetFungibleSupportedChains) {
 TEST_F(AssetDiscoveryManagerUnitTest, GetNonFungibleSupportedChains) {
   // Gnosis chain ID should not be included if it's not a custom network
   auto chains = asset_discovery_manager_->GetNonFungibleSupportedChains();
-  EXPECT_EQ(chains.size(), 7UL);
+  EXPECT_EQ(chains.size(), 8UL);
```

Comment:
why did that change?

Reply:
Since I added Base to the list of BalanceScanner supported networks.